controller/network/debug.go

```diff
@@ -0,0 +1,78 @@
+package network
```

Comment:
needs header and a comment as to what this is all about. did you commit this intentionally? feels out of place - thus maybe a comment?

Reply:
Renamed to util_test and added a comment. Added some other potentially common test bits there as well